DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/src/main.cpp

```diff
@@ -14,12 +14,12 @@
 #include "anr.hpp"
 #include "anr_params.hpp"
 #include "constants.hpp"
+#include "constexpr_math.hpp"
```

Comment:
Is this header actually used?  I see no other changes in this file, I would have thought you'd have to update the namespace if any functions from this file were actually used here.